README.md

```diff
@@ -4,10 +4,10 @@
 https://gitter.im/TypeStrong/atom-typescript](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/TypeStrong/atom-typescript?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 JavaScript developers can now just open a `.ts` file and start hacking
-away like they are used to. No `grunt` no `Visual Studio`. Just pure
+away like they are used to. No `grunt`, no `Visual Studio`. Just pure
 coding.
 
-**NOTE**: When updating to **v12**, make sure `language-typescript` core
+**NOTE**: When updating to **v12**, make sure the `language-typescript` core
```

Comment:
Hmm. I'm not a native English speaker, but does English grammar require the definite article here? I believe in this context `language-typescript` can be interpreted as the proper name of the package, and from what I remember from my English lessons, proper names do not require the definite article, do they?